pimd/pim_rp.c

```diff
@@ -431,26 +433,20 @@ int pim_rp_new(struct pim_instance *pim, struct in_addr rp_addr,
 	struct pim_upstream *up;
 	bool upstream_updated = false;
 
-	if (rp_addr.s_addr == INADDR_ANY ||
-	    rp_addr.s_addr == INADDR_NONE)
```

Comment:
Hmm, this seems to have gotten mixed in from the `NONE` ⇒ `ANY` change. Should merge that first I guess, to keep things in correct order.